src/lobby/react.js

```diff
@@ -263,7 +182,7 @@ class Lobby extends React.Component {
       return;
     }
 
-    let multiplayer = undefined;
+    let multiplayer = null;
```

Comment:
I think there are `multiplayer === undefined` checks that trigger singleplayer mode, so you will break that by changing this to `null` (which was originally fixed in https://github.com/nicolodavis/boardgame.io/commit/d7d6b445cdf9d1bd740bbf4e1e7c139c6ffcc089).

Reply:
oh, thanks, I'll revert it back. But, in general, that's a bit strange..